rate: 100
gravity: 9.81

Kp_pos: [35.0, 35.0, 50.0]
Kd_pos: [50.0, 50.0, 50.0]
Ki_pos: [5.0, 5.0, 10.0]

Kp_att: [15.0, 15.0, 15.0]
Kd_att: [2.0, 2.0, 2.0]

mass: 1.5
Ixx: 0.0347563    
Iyy: 0.0458929
Izz: 0.0977
Kf: 8.54858e-06
Kq: 1.6e-2
r_com_off_x: 0.0
r_com_off_y: 0.0
r_com_off_z: 0.0
arm: 0.215
#ENU
# rotor_angle: [-0.7853, 0.7853, 2.356194, -2.356194, 0.7853, -0.7853, -2.356194, 2.356194]
# rotor_sign: [1, -1, 1, -1, 1, -1, 1, -1] #1 ccw -1cw
#NED
rotor_angle: [-0.52359877559, -1.57079632679, -2.61799387799, 2.6179938779, 1.5707963267, 0.5235987755]
rotor_sign: [-1, 1, -1, 1, -1, 1] #-1 ccw 1cw 

arm_height: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]